system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1080,6 +1059,11 @@ pub type Migrations = (
 	frame_support::migrations::RemovePallet<DmpQueueName, RocksDbWeight>,
 	pallet_collator_selection::migration::v2::MigrationToV2<Runtime>,
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
+	pallet_assets::migration::next_asset_id::SetNextAssetId<
```

Comment:
@joepetrowski still needs to delete `MAX_PRIME` :P

Reply:
Done :)
https://assethub-kusama.subscan.io/extrinsic/7559849-2

Reply:
j6A5^$wadK$8);n